packages/core/src/components/radio-pill/_radio-pill.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../global/variables';
+
+@include exports('ray-radio-pill') {
+  .#{$ray-class-prefix}radio-pill__wrapper {
+    width: 100%;
+    display: flex;
+  }
+
+  .#{$ray-class-prefix}radio-pill {
+    float: left;
+    display: flex;
+    flex-grow: 1;
```

Comment:
can remove both `flex-grow: 1;` and `float: left` here I believe

Reply:
<img width="268" alt="Screen Shot 2019-05-07 at 5 06 22 PM" src="https://user-images.githubusercontent.com/17712022/57333301-7f2fad80-70ea-11e9-86e8-55d67cee6d45.png">
hmm without those it looks wrong